code/game/jobs/job/civilians/support/synthetic.dm

```diff
@@ -50,3 +50,42 @@
 	name = JOB_SYNTH
 	icon_state = "syn_spawn"
 	job = /datum/job/civilian/synthetic
+
+
+// ship synthetic
+
+/datum/job/civilian/synthetic/ship
+	title = JOB_SHIP_SYNTH
+	total_positions = 1
```

Comment:
`/datum/job/civilian/synthetic` must then be reduced to 1

Reply:
Likely also need to reduce `/proc/synth_slot_formula(playercount)` 's `max` argument from 2 to 1.